app/routes/_base.tsx

```diff
@@ -20,11 +18,9 @@ export const meta = ({ error }: { error: ErrorResponse }) => {
 export default function BaseLayout() {
   return (
     <>
-      <Header />
```

Comment:
It does not make sense to have layout like this
We should remove layout at all if we don't need it, but investigation is required
If we still need it you should check if we can use it without naming every file with _base